code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -1,3 +1,5 @@
+// Drakeborg sprites by Mizartz. drakeborg are licensed under the CC BY-NC-SA 3.0 license. licenseing infomation located in \icons\mob\cyborg
+
```

Comment:
delete this

Reply:
took silicons suggestion by instead of seven comments just make one mentioning the credit & the licence on the top.